manifest.xml

```diff
@@ -15,7 +15,7 @@
 
     <requires>
         <type>elgg_release</type>
-        <version>2.2.3</version>
+        <version>2.2</version>
```

Comment:
Leave this as is. The only change that requires 2.2 is elgg_view_input which is unnecessary